gulpfile.js

```diff
@@ -143,9 +143,7 @@ var jsTasks = function(filename) {
 var writeToManifest = function(directory) {
   return lazypipe()
     .pipe(gulp.dest, path.dist + directory)
-    .pipe(function() {
-      return gulpif('**/*.{js,css}', browserSync.reload({stream:true}));
-    })
+    .pipe(browserSync.stream, {match: '**/*.{js,css}'})
```

Comment:
Can you compare my [bs_files-only](https://github.com/joemaller/sage/tree/bs_files-only) branch with the code you're seeing running? I'm only seeing full reloads from that, no CSS injection and a ton of changed file notices.